lint.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+const https = require("https");
+const fs = require("fs");
+const util = require("util");
+
+const get = url =>
+    new Promise((resolve, reject) => {
+        https.get(url, res => {
+            res.setEncoding("utf8");
+            let body = "";
+            res.on("data", data => {
+                body += data;
+            });
+            res.on("end", () => {
+                resolve(body);
+            });
+            res.on("error", err => {
+                reject(err);
+            });
+        });
+    });
+
+const readFile = util.promisify(fs.readFile);
+
+const THEME_COLOR_REFERENCE_URL =
+    "https://code.visualstudio.com/docs/getstarted/theme-color-reference";
+
+const NOT_THEME_KEYS = [
+    "workbench.colorCustomizations",
+    "editor.tokenColorCustomizations"
+];
+
+const fileToLint = process.argv[2];
+
+if (!fileToLint) {
+    throw new Error("You need to specifiy the path of the theme to lint");
+}
+
+async function scrapThemeAvailableKeys() {
+    const data = await get(THEME_COLOR_REFERENCE_URL);
+
+    const matches = data.match(new RegExp("<code>.+?</code>", "g"));
+
+    const availableKeys = [...matches]

```

Comment:
yowza! Really muscling this one here.. Assuming this works, but might be fragile.